scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -10,10 +10,13 @@
 ACCEPTED_ENCODINGS = [b'gzip', b'deflate']
 
 try:
+    flag=0
     import brotli
     ACCEPTED_ENCODINGS.append(b'br')
 except ImportError:
     pass
+else:
+    flag=1
```

Comment:
I don’t think you need a new variable here, you could just use `b'br' in ACCEPTED_ENCODINGS` to the same effect, which would be more readable.